src/app/api/api_v1/endpoints/robot_type.py

```diff
@@ -0,0 +1,76 @@
+from fastapi import APIRouter, Depends, HTTPException
+
+from app.api import deps
+from app.crud.crud_robot_type import *
+from app.schemas.robot_type import *
+
+router = APIRouter()
```

Comment:
For the sake of not indicating a closer grouping of imports with this line than with the top level functions here we should have two newlines between imports and this statement